processor/eval/ledger_for_evaluator.go

```diff
@@ -148,3 +148,8 @@ func (l LedgerForEvaluator) LatestTotals() (ledgercore.AccountTotals, error) {
 	_, totals, err := l.Ledger.LatestTotals()
 	return totals, err
 }
+
+// BlockHdrCached is part of go-algorand's indexerLedgerForEval interface.
```

Comment:
I don't see where else in the PR this function is used. Was this brought in by the `go-algorand` update? Is there a way to know that this is implemented correctly (seems like it should since this is a pass-through) ?

Reply:
I should also mention that if indexer is not actually making use of this method, I would argue for implementing via `panic("not implemented")` instead.

Reply:
Yeah it was added in https://github.com/algorand/go-algorand/commit/b8007e4ade943d4bc181a301d1291685eaf60fc4#diff-69a9375fb82e9e6878855f6e962f7b3eeba8bf108dc6c2c07ab21e35cfb8d267R47

And no we don't call it in our code, but it has to be defined as part of the interface, so I just copied what I saw in the above commit.

Reply:
I'm actually not confident that it's not used. It seems to be called inside `eval` in b8007e4ade943d4bc181a301d1291685eaf60fc4 and as you've pointed out, there wouldn't seem to be a reason to add it to the interface if it wasn't being used.